evento/app/controllers/users_controller.rb

```diff
@@ -63,7 +64,11 @@ def set_user
   end
 
   # Only allow a trusted parameter "white list" through.
-  def user_params
-    params.require(:user).permit(:name, :email, :password)
+  def create_params
```

Comment:
Hmm after further inspection I noticed that `request.body` is
`name=NewName` 
when 
`put "/users/#{user.id}", params: { name: "NewName" }, headers: auth_header` 
is called,

`user[name]=NewName` 
when 
`put "/users/#{user.id}", params: { user: { name: "NewName" } }, headers: auth_header` 
is called,

`{"name":"NewName"}` 
when 
`put "/users/#{user.id}", params: { "name": "NewName"}.to_json, headers: { "Authorization": json["auth_token"], 'CONTENT_TYPE' => 'application/json' }` 
is called.

And the last two work with `user_params`


So what should we do? It's fine for me if we decide to only allow requests with json body.